bowling.js

```diff
@@ -0,0 +1,120 @@
+class Bowling {
+  constructor() {
+    this.frame = 0;
+    this.totalScore = 0;
+    this.frames = [];
+    this.scoreFrames = [];
+    this.roll = 1;
+    this.isStrike = false;
+  }
+
```

Comment:
All methods set out well and are only doing one thing. (Single responsibility principle).